src/main/java/uk/gov/hmcts/reform/sandl.snlevents/controllers/SessionController.java

```diff
@@ -81,6 +81,14 @@ public ResponseEntity insertSession(@RequestBody CreateSession createSession) th
         return ok(ut.getId());
     }
 
+    @PutMapping(path = "/update", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity updateSession(@RequestBody UpsertSession upsertSession) throws IOException {
+
+        UserTransaction ut = sessionService.updateSession(upsertSession);
```

Comment:
What about a call to rules?

Reply:
It's within sessionService.updateSession method